README.md

```diff
@@ -56,10 +58,50 @@ Vue.use(VueKeyCloak)
 Vue.use(VueKeyCloak, options)
 ```
 
+### Vue 3
+
+> `createApp(App).use(VueKeycloak, [options])`
+
+Tell Vue to install the plugin, and optionally pass in a JavaScript object additional
+configuration.
+
+```javascript
+import VueKeyCloak from '@dsb-norge/vue-keycloak-js'
+
+createApp(App).use(VueKeycloak)
+
+// You can also pass in options. Check options reference below.
+createApp(App).use(VueKeycloak, options)
+```
+
+```typescript
+import VueKeyCloak from '@dsb-norge/vue-keycloak-js'
+import { VueKeycloakInstance } from "@dsb-norge/vue-keycloak-js/dist/types";
+
+createApp(App).use(VueKeycloak)
+
+// You can also pass in options. Check options reference below.
+createApp(App).use(VueKeycloak, options)
+
+// Allow usage of this.$keycloak in components
+declare module '@vue/runtime-core' {
+  interface ComponentCustomProperties  {
+    $keycloak: VueKeycloakInstance
+  }
+}
+```
+
 The plugin adds a `$keycloak` property to the global Vue instance.
-This is actually a [Vue.observable](https://vuejs.org/v2/api/#Vue-observable) instance and can be used as such.
-It shadows most of the keycloak instance's properties and functions. All other variables & functions can be found
-on `$keycloak.keycloak` attribute
+This shadows most of the keycloak instance's properties and functions.
+All other variables & functions can be found on `$keycloak.keycloak` attribute
+
+Internally for Vue 2:
+ * [Vue.observable](https://vuejs.org/v2/api/#Vue-observable)
+
+Internally for Vue 3
+* [reactive](https://v3.vuejs.org/guide/reactivity-fundamentals.html#declaring-reactive-state)
+
+This object is reactive and will update with new tokens and other information
```

Comment:
If this is not yet reactive, then perhaps either fix root cause or change readme?

Reply:
It is reactive in v2 and should be in v3